app/src/ui/open-pull-request/open-pull-request-dialog.tsx

```diff
@@ -179,25 +193,42 @@ export class OpenPullRequestDialog extends React.Component<IOpenPullRequestDialo
   }
 
   private renderFooter() {
-    const { mergeStatus, commitSHAs } = this.props.pullRequestState
-    const gitHubRepository = this.props.repository.gitHubRepository
+    const { currentBranchHasPullRequest, pullRequestState, repository } =
+      this.props
+    const { mergeStatus, commitSHAs } = pullRequestState
+    const isHostedOnGitHub = isRepositoryHostedOnGitHub(repository)
```

Comment:
Can we use the existing type guard `isRepositoryWithGitHubRepository` here instead?